packages/compass-import-export/src/import/import-csv.ts

```diff
@@ -22,7 +25,7 @@ type ImportCSVOptions = {
   dataService: DataService;
   ns: string;
   input: Readable;
-  output: Writable;
+  output?: Writable;
```

Comment:
This is the stream where the log stuff goes. Making it optional for now because the modal doesn't have a place to access it anyway. That makes integration in smaller steps easier.